roles/neutron-data-network/tasks/main.yml

```diff
@@ -59,6 +59,45 @@
     neutron-metadata-agent:
       config_files: /etc/neutron/metadata_agent.ini
 
+- name: neutron lbaas config
+  template:
+    src=etc/neutron/neutron_lbaas.conf
+    dest=/etc/neutron/neutron_lbaas.conf
+    mode=0644
+  notify:
+    - restart neutron services
+  when: neutron.lbaas.enable
+
+- name: neutron lbaas agent config
+  template:
+    src=etc/neutron/services/loadbalancer/haproxy/lbaas_agent.ini
+    dest=/etc/neutron/services/loadbalancer/haproxy
+    mode=0644
+  notify:
+    - restart neutron services
+  when: neutron.lbaas.enable
+
+- name: install neutron lbaas service
+  upstart_service:
+    name: "{{ item.key }}"
+    user: neutron
+    cmd: "/usr/local/bin/{{ item.key }}"
+    pidfile: "{{ item.value.pidfile|default('') }}"
```

Comment:
If the default is blank, can the argument be omitted? (thus using the `omit` filter?)